.github/workflows/build.yml

```diff
@@ -340,3 +369,44 @@ jobs:
       - name: copy binaries to s3
         run: |
           aws s3 sync . s3://apex-net/
+
+  build-rpm:
+    needs: ["go-lint"]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: docker/setup-buildx-action@v2
+      - name: Install make
+        run: |
+          sudo apt -y install make
+      - name: Build rpm
+        run: |
+          make rpm
```

Comment:
Is it also worth adding the RPMs as artifacts to the CI job? That way if we could grab RPMs to test specific PRs which might be cool

Reply:
Yeah, that's a good idea. I'll add it.

For building the full rpm, I only test building it once (for Fedora 37), but I have copr do multiple builds for different distros/versions (Just F37 and F38 so far).

So we'd just have the F37 rpm in artifacts, but source rpms for every distro we publish for on copr.

Reply:
rpms from `build-rpm` stored as artifacts now